src/pages/Vaults/VaultsOverview/components/CreateVaultWizard/CreateVaultWizard.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'react';
+
+import { VaultsTableRow } from '../VaultsTable';
+import DespositCollateralStep from './DespositCollateralStep';
+import DisclaimerStep from './DisclaimerStep';
+import { Steps } from './Step';
+import VaultCreatedStep from './VaultCreatedStep';
+
+type Props = {
+  vault?: VaultsTableRow;
+};
+
+type CreateVaultWizardProps = Props;
+
```

Comment:
This seems a bit redundant :) is there a reason for reassigning the Props type? Also, is there a reason for using a type rather than an interface? We should be consistent with this when creating components.

Reply:
here it can be an `interface` but for more complex type its beter to just `types`